deployment-manager/src/main/java/com/example/deploymentmanager/DeploymentManagerDemo.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.deploymentmanager;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.deploymentmanager.DeploymentManager;
+import com.google.api.services.deploymentmanager.model.*;
+
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class DeploymentManagerDemo {
+
+  private static String CONFIG_YAML = "yaml/deployment-manager-config.yaml";
+  private static String DEPLOYMENT = "my-deployment-1";
+
+  public static void main(String args[]) throws GeneralSecurityException, IOException {
+
+    String GOOGLE_PROJECT_ID = System.getenv("GOOGLE_PROJECT_ID");
+    if (GOOGLE_PROJECT_ID == null || GOOGLE_PROJECT_ID.compareTo("") == 0) {‰
+      System.out.println("GOOGLE_PROJECT_ID is not set");
+      System.exit(-1);
+    }
+
+    // best practice is to use application default credential. If inside a GCE
+    // instance it is set automatically.
+    
+    GoogleCredential credential = GoogleCredential.getApplicationDefault();
+
+    // to load from somewhere else use this
+    // GoogleCredential credential = GoogleCredential.fromStream(new
+    // FileInputStream(path_to_key));
+
+    HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+    JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
+
+    DeploymentManager dm2 = new DeploymentManager.Builder(httpTransport, jsonFactory, credential)
+        .setApplicationName("New_Application").build();
+
+    Deployment deploy = new Deployment();
+    // Optional we want some labels: Map of labels; provided by the client when the
+    // resource is created or updated.
+    List<DeploymentLabelEntry> labels = new ArrayList<DeploymentLabelEntry>();
+
```

Comment:
Can you group up some of these statements a bit more? The empty lines between statements seem unnecessary.